.github/CODEOWNERS

```
@@ -27,8 +25,9 @@
 /package-lock.json @kbdharun @sebastiaanspeck
 /requirements.txt @kbdharun @sebastiaanspeck
 
-/contributing-guides/maintainers-guide.md @sbrl @kbdharun
-/contributing-guides/style-guide.md @sbrl @kbdharun
+/contributing-guides/maintainers-guide.md @sbrl @kbdharun @sebastiaanspeck
+/contributing-guides/style-guide.md @sbrl @kbdharun @sebastiaanspeck
+/contributing-guides/translation-templates/ @sebastiaanspeck
```

```suggestion
/contributing-guides/translation-templates/ @kbdharun @sebastiaanspeck
```
If possible, can you add me for translation templates too?